src/calculateRentalCost.js

```diff
@@ -4,7 +4,13 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
+  } else if (days >= 3) {
+    return days * 40 - 20;
```

Comment:
Remove the `else` keyword as it is unnecessary after an `if` block that contains a `return` statement. Simplify the control flow as described in checklist items 4 and 5.